### ISSUE TITLE
Issue with NGINX assets

### ISSUE THREAD

**Issue by Olivier6767** [dvershinin/lastversion#24]
Hello, when trying to get the latest version of nginx, there is an issue with retrieving the assets:

**lastversion --format assets nginx**
```
Traceback (most recent call last):
  File "/usr/bin/lastversion", line 8, in <module>
    sys.exit(main())
  File "/usr/lib/python3.8/site-packages/lastversion/lastversion.py", line 246, in main
    res = latest(args.repo, args.format, args.pre, args.filter,
  File "/usr/lib/python3.8/site-packages/lastversion/lastversion.py", line 92, in latest
    return s.get_assets(release, short_urls, assets_filter)
AttributeError: 'MercurialRepoSession' object has no attribute 'get_assets'
```

Same issue when using --format json
Is this something that can be fixed?

Thanks

**Comment by dvershinin** [comment]
Hi, 

You can use `lastversion nginx --format source` instead.
For non-Github-hosted projects, assets and sources would probably be synonyms anyway...

**Comment by dvershinin** [comment]
Thank you for the bug report. Fixed in v1.2.2
